src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCacheOptions.cs

```diff
@@ -35,6 +35,13 @@ public class MemoryCacheOptions : IOptions<MemoryCacheOptions>
         /// <summary>
         /// Gets or sets the maximum size of the cache.
         /// </summary>
+        /// <remarks>
+        /// The units are arbitrary. Users specify the size of every entry
+        /// they add to the cache. If no size is specified, the entry has 
+        /// no size and the size limit is ignored for that entry.
+        /// For more information, see
+        /// <see href="https://learn.microsoft.com/aspnet/core/performance/caching/memory#use-setsize-size-and-sizelimit-to-limit-cache-size-1">Use SetSize, Size, and SizeLimit to limit cache size</see>.
```

Comment:
The link isn't anchored to the specified section correctly.
```suggestion
        /// <see href="https://learn.microsoft.com/aspnet/core/performance/caching/memory#use-setsize-size-and-sizelimit-to-limit-cache-size">Use SetSize, Size, and SizeLimit to limit cache size</see>.
```